indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -169,7 +169,8 @@
     {% set can_reset = registration.registration_form.moderation_enabled and not registration.is_paid %}
     {% set ticket_template = registration.registration_form.get_ticket_template() %}
     {% set can_get_ticket = registration.registration_form.tickets_enabled and (not ticket_template.is_ticket or not registration.is_ticket_blocked) %}
-    {% if can_reset or can_get_ticket %}
+    {% set can_get_document = true %}
+    {% if can_reset or can_get_ticket or can_get_document %}
```

Comment:
should we always allow users to download document or should we have a flag like `tickets_enabled`?

Reply:
This has nothing to do with tickets. Use the same logic that's also use in the registrant list to determine whether the "generate documents" dropdown item is enabled or not.